coderd/workspaceagentsrpc.go

```diff
@@ -132,7 +132,7 @@ func (api *API) workspaceAgentRPC(rw http.ResponseWriter, r *http.Request) {
 		WorkspaceID:    workspace.ID,
 		OrganizationID: workspace.OrganizationID,
 
-		Ctx:                               api.ctx,
+		Ctx:                               ctx,
```

Comment:
I think this context is only used in `startCacheRefreshLoop`?

If so, we can rename `Ctx` to like `AuthenticatedCtx` or something akin to that? So users of it know what it carries?